ThingIFSDK/ThingIFSDK/ThingIFAPI.swift

```diff
@@ -87,6 +87,7 @@ public class ThingIFAPI: NSObject, NSCoding {
     /** On board IoT Cloud with the specified vendor thing ID.
     Specified thing will be owned by owner who consumes this API.
     (Specified on creation of ThingIFAPI instance.)
+    If you are using a gateway, you need to use onboardEndnodeWithGateway instead.
```

Comment:
better to describe like this: 
`If you are using a gateway, you need to use onboardEndnodeWithGateway to onboard endnode instead.`